core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer_utils.js

```diff
@@ -0,0 +1,240 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
```

Comment:
Why is this file named differently than the other utils? I would expect _puppeteerUtils.js_, or maybe the naming should be _user-factory.js_, ... as we use that elsewhere.

Reply:
Still the utils are named differently than the suites, is that expected? If yes, why?

Reply:
I think we can use file name convention everywhere as: `show-message-utils.js` or `puppeteer-utils.js` or `user-factory.js`. @vojtechjelinek  should I change according to this?

Reply:
I think so

Reply:
done